### PR TITLE
[Australia]: CFP - tickets reminder

### PR DESCRIPTION
**Pull request by swapnilogale**
----
📚 Documentation preview 📚: https://writethedocs-www--2185.org.readthedocs.build/

### CODE REVIEW

**Comment by swapnilogale** [comment]
@plaindocs - reminder for CFP and grants. Good to go.

File: docs/conf/australia/2024/news/tickets-cfp-reminder.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+:template: {{year}}/generic.html
+
+.. post:: August 6, 2024
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
+
+Ticket Sales, Grant Program, and Call for Proposals Reminder
+=============================================================
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/tickets-cfp-reminder.rst]: I'd prefer this to be in the order of topics (and I think leading with CFP is correct)

File: docs/conf/australia/2024/news/tickets-cfp-reminder.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Greetings, documentarians! Just a quick reminder that tickets are `on sale now <https://www.writethedocs.org/conf/australia/{{year}}/tickets/>`_, and to remind you that our `Call for Proposals <https://www.writethedocs.org/conf/australia/{{year}}/cfp/>`_ is still open.
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/tickets-cfp-reminder.rst]: Ditto, and it's odd that this doesn't have grant program :-p

**Review by plaindocs — APPROVED** [review]
Couple of suggestions only.

**Pull request merged**